sites/public/pages/eligibility/income.tsx

```diff
@@ -84,16 +85,18 @@ const EligibilityIncome = () => {
             <p className="field-note mb-4" id="income-description">
               {t("eligibility.income.description")}
             </p>
-            <Select
+            <Field
               id="income"
               name="income"
               label={t("eligibility.income.label")}
               describedBy="income-description"
+              defaultValue={eligibilityRequirements.income}
+              inputProps={{ maxLength: 7 }}
+              type={"number"}
               validation={{ required: true }}
+              error={errors.income}
+              errorMessage={t("errors.numberError")}
               register={register}
-              controlClassName="control"
-              options={incomeRanges}
-              keyPrefix="eligibility.income.ranges"
             />
```

Comment:
I think it would be helpful to include the placeholder prop here, so user's have an idea how to input the income.  We don't want people to include the `$` so leveraging placeholder is helpful.  For example:
`placeholder={"XX,XXX"}`

Reply:
Oh, the input doesn't allow you to type anything other than numbers (so it doesn't allow characters like "$" or ",").